pkg/config/config.go

```diff
@@ -278,7 +284,7 @@ func (c Config) GetExcludesAsRegularExpression() string {
 // Save saves the config to it's Path
 func (c Config) Save(version string) error {
 	if utils.IsRegularFile(c.Path) {
-		return fmt.Errorf("File `%v` already exists", c.Path)
+		return fmt.Errorf("file `%v` already exists", c.Path)
```

Comment:
There is no reason to use %v for a string

So, this would be preferred

```suggestion
		return fmt.Errorf("file `%s` already exists", c.Path)
```

Also, this could be considered


```suggestion
		return fmt.Errorf("file %q already exists", c.Path)
```

Reply:
I decided to keep the quotes - if for some obscure reason `c.Path` is empty or only filled with unprintable characters one would no longer recognize that this error message was supposed to contain a path.
But I replaced the default rendering with string rendering.